README.adoc

```diff
@@ -167,6 +167,31 @@ Where :
 
 Beware : you will have concurrency issues if multiple containers are running on this single volume.
 
+=== Running integration tests against your James container
+
+We wrote some MPT (James' Mail Protocols Tests subproject) integration tests to validate a James 
+deployement.
+
+It uses the External-James module, that can easily use environment variables to locate a distant 
+IMAP server and run integration tests against it.
+
+For that, you need to handle the domain +domain+ and configured user +imapuser+ with password 
++passord+. See above documentation to see how you can do this.
+
+You may want to run MPT tests inside docker. As you need to use maven, the simplest option is to 
```

Comment:
I thought to it, but didn't write it as :
1/ There is nothing about how to build and deploy James the old way
2/ To run these tests you need MPT and Mailbox and all there dependencies with correct version in your maven home...